### PR TITLE
*: pair with sqlboiler-psql driver impl.

### PR DESCRIPTION
**Pull request by glerchundi**

### CODE REVIEW

File: driver/crdb.go [hunk 1]
```diff
@@ -447,8 +453,13 @@ func getArrayType(c drivers.Column) string {
...
 
-	col.TestSingleton = importers.Map{
-		"crdb_upsert": {
+	col.All = importers.Set{
+		Standard: importers.List{
+			`"strconv"`,
+		},
+	}
+	col.Singleton = importers.Map{
+		"psql_upsert": {
```
Comment (aarondl) [inline driver/crdb.go]: I assume you didn't mean for this to change to psql :)

File: driver/override/templates/17_upsert.tpl [hunk 2]
```diff
@@ -0,0 +1,161 @@
+{{- $alias := .Aliases.Table .Table.Name}}
```
Comment (aarondl) [inline driver/override/templates/17_upsert.tpl]: Due to a change I made yesterday afternoon in the v3 branch, templates now require a file extension that defines them as both their normal type and a template. This allows sqlboiler to generate any kind of file, not just go.

You'll have to rename this to 17_upsert.go.tpl and same to the rest of the templates.

File: driver/crdb.go (hunk 1 above)
Comment (glerchundi) [reply driver/crdb.go]: Ups! 👍

File: driver/override/templates/17_upsert.tpl (hunk 2 above)
Comment (glerchundi) [reply driver/override/templates/17_upsert.tpl]: Ok, thanks.

**Comment by glerchundi** [comment]
@aarondl I need to do a full e2e test and will finish tomorrow but i'm really close to goal!

**Comment by aarondl** [comment]
Good to hear. I'm diligently working on the videos and what not. Getting quite a bit done but still a ways out. I merged your pull request btw. Untagged currently.

**Comment by glerchundi** [comment]
Merging!

**Pull request merged**